strings.txt

```diff
@@ -403,3 +403,17 @@ PLUGIN_QOBUZ_ALBUM_NOT_FOUND
 	DE	Album nicht in der Qobuz-Bibliothek gefunden
 	NL	Album niet gevonden in de Qobuz bibliotheek
 	FR	Album introuvable dans la bibliothèque Qobuz
+	
+QOBUZ_CLEAR_SEARCH_HISTORY
```

Comment:
I'm sorry to be a pain... but by convention we should be using `PLUGIN_QOBUZ_...`.